src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,28 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, user, body, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <p>
+          {'Posted by '}
+
+          <UserInfo user={user} />
```

Comment:
<img width="489" alt="image" src="https://github.com/user-attachments/assets/908c1df9-49aa-407f-a715-cdcad321aec0">
find can return undefined, so it's better to add the check
```suggestion
          {user && <UserInfo user={user} />}
```